.coveragerc

```diff
@@ -0,0 +1,4 @@
+[report]
```

Comment:
This should really move into pyproject. https://github.com/abn/cookiecutter-python-project/blob/732bf8e890da9847dca407d2f30f254fd963bdca/%7B%7B%20cookiecutter.directory_name%20%7D%7D/pyproject.toml#L78-L125

Reply:
Done!